app/controllers/landing_page_controller.rb

```diff
@@ -5,9 +5,16 @@ class LandingPageController < ContentItemsController
 
   # SCAFFOLDING: can be removed when basic content items are available
   # from content-store
-  def request_content_item(_base_path)
-    GdsApi.content_store.content_item(request.path).to_h
-  rescue StandardError
+  def content_item
+    @content_item ||= ContentItemFactory.build(old_scaffolding_content_item)
+  end
+
+  # SCAFFOLDING: can be removed when basic content items are available
+  # from content-store
+  def old_scaffolding_content_item
+    result = ContentItemLoader.load(request.path)
```

Comment:
It's great that you've fixed this. I think it would be good to add a test for this scaffolding too.

Reply:
They can't have been thorough enough because nothing caught the attempted call to `ContentItemLoaderGdsApi.content_store.content_item(request.path).to_h` which doesn't exist!